README_BUILD.md

```diff
@@ -27,18 +27,39 @@ rake dsc:types:build       # Build DSC types in (lib/puppet/type)
 rake dsc:types:clean       # Cleanup DSC types in (lib/puppet/type)
 ~~~
 
-The default task 'bundle exec rake' will run all of this tasks.
-
 ## Building
 
 When building the types and specs, keep the following considerations in mind:
 
-* Resources must be built with a non-Windows box due to limitations in the gems that the builder uses to generate types and specs.
-* You must use Ruby 1.9.3 to build the types (again due to a limitation in the builder dependencies).
-* The builder requires a MOF file and a PSD1 to build a type. Those need to be present for it to be successful.
-* The MOF file encoding should be UTF-8. This ensures the most compatibility with the mof gem.
-* If you do a DSC build from a fresh clone, it will build the latest DSC resources, which may or may not be what you want to do. To avoid this, you should set the environment variable `DSC_REF` to the SHA1 of https://github.com/PowerShell/DscResources commit that you want to build Puppet DSC resource types from.
-* When updating the Puppet DSC resource types and specs, please include the SHA1 commit that the types were built from in your git commit message. This makes it easier to track down exactly what version of `DscResources` we have built against.
+Prerequisites
```

Comment:
Hmm, either make this an H3, or remove it? It's a little redundant with the lead in above.